text/0043-disable-sbom.md

```diff
@@ -0,0 +1,50 @@
+# Provide Global Mechanism to Disable SBOM Generation
+
+## Summary
+
+Users should have a mechanism to disable the generation of SBOM documents
+during the build process.
+
+## Motivation
+
+The generation of the SBOM documents for an image can be very time-intensive in
+some cases. We have seen a doubling of build times for Node.js applications.
+Additionally, some users are not interested in taking advantage of the SBOM
+features of the buildpacks.
+
+## Detailed Explanation
+
+If a user wishes to produce an image that does not contain SBOM documents, they
+can set the `BP_GENERATE_SBOM` environment variable to `false`. When unset, the
+value of this environment variable is assumed to be `true`.
+
+When `BP_GENERATE_SBOM` is set to `false`, none of the buildpacks executing
```

Comment:
In the Buildpacks Spec v0.7/ Platform Spec v0.8, the `restorer` phase of the lifecycle [restores SBOMs from previous builds](https://github.com/buildpacks/spec/blob/buildpack/v0.7/platform.md#outputs-2). What's the expected behaviour of turning off the SBOM with this env var if there are preexisting SBOM files from earlier builds?

Reply:
If the layer is reused, then we could say that the behavior is that previously generated SBOM outputs may survive into the new image. That is to say, it is not on buildpacks to clear out SBOMs from previous builds. Ultimately, in most cases, the SBOM should be cleared as soon as the layer needs to be rebuilt for a new package or runtime version. Or, the user could clear their cache before the build to ensure a fresh start.

Reply:
Addressed in https://github.com/paketo-buildpacks/rfcs/pull/161/commits/c566e4612de413b399b20e9d4e2882011aab7883